### PR TITLE
Use current parameters

### LINKED ISSUE #44: Use policy as of the current date
e.g. for UC

### LINKED ISSUE #151: Use policy parameters as of today's date

### PR DESCRIPTION
**Pull request by nikhilwoodruff**
This PR:
- Uses the recent current parameter reform to use current parameters
- <del>Adds the employer NI rate</del>
- <del>Includes employer NI when calculating the budgetary impact</del>

### CODE REVIEW

File: server/situations/charts.py [hunk 1]
```diff
@@ -79,7 +80,7 @@ def mtr_chart(baseline, reformed):
 def household_waterfall_chart(reform, labels, situation, baseline, reformed):
     net_income = [baseline.calc("net_income").sum()]
     for i in range(1, len(reform)):
-        partially_reformed = situation(IndividualSim(reform[:i]))
+        partially_reformed = situation(IndividualSim(reform[:i], year=2021))
```
Comment (MaxGhenis) [inline server/situations/charts.py:83]: could you make this dynamic to the current year, or file an issue to do so later if it's complicated?

File: client/src/js/pages/policy/default_policy.js [hunk 2]
```diff
@@ -44,8 +44,8 @@ const DEFAULT_POLICY = {
 	personal_allowance: {
 		title: "Personal allowance",
 		description: "The personal allowance is deducted from general income.",
-		default: 12500,
-		value: 12500,
+		default: 12570,
```
Comment (MaxGhenis) [inline client/src/js/pages/policy/default_policy.js:47]: could you file an issue to have this pull from current parameter values if that can't be done here?

**Review by MaxGhenis — CHANGES_REQUESTED** [review]
please also update https://github.com/UBICenter/policyengine-uk/blob/master/client/public/faq.md#as-of-what-date-are-policy-parameters-set

after that and these suggestions, good to merge

File: server/situations/charts.py (hunk 1 above)
Comment (nikhilwoodruff) [reply server/situations/charts.py:83]: Filed

File: client/src/js/pages/policy/default_policy.js (hunk 2 above)
Comment (nikhilwoodruff) [reply client/src/js/pages/policy/default_policy.js:47]: Filed

**Pull request merged**